pkg/util/testing/wrapper.go

```diff
@@ -506,33 +620,34 @@ func MakeTrainingRuntimeWrapper(namespace, name string) *TrainingRuntimeWrapper
 												Containers: []corev1.Container{
 													{
 														Name: constants.ContainerDatasetInitializer,
-														VolumeMounts: []corev1.VolumeMount{
-															jobsetplugin.VolumeMountDatasetInitializer,
-														},
+														VolumeMounts: []corev1.VolumeMount{{
+															Name:      jobsetplgconsts.VolumeNameInitializer,
+															MountPath: constants.DatasetMountPath,
+														}},
 													},
 													{
 														Name: constants.ContainerModelInitializer,
-														VolumeMounts: []corev1.VolumeMount{
-															jobsetplugin.VolumeMountModelInitializer,
-														},
+														VolumeMounts: []corev1.VolumeMount{{
+															Name:      jobsetplgconsts.VolumeNameInitializer,
+															MountPath: "constants.ModelMountPath",
```

Comment:
I think, there are a few places where you accidentally replaced this value.
```suggestion
															MountPath: constants.ModelMountPath,
```

Reply:
Oh, good catch...

Reply:
Done.